client/src/component/CohortTrainees.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+
+export const TraineesTable = () => {
+  const [name, setName] = useState("");
+  const [trainees, setTrainees] = useState([]);
+
+  const handleNameChange = (event) => {
+    setName(event.target.value);
+  };
+
+  const handleAddTrainee = (event) => {
+    event.preventDefault(); // Prevent of loading page
+    if (name.trim() !== "") {
+      setTrainees([...trainees, name]);
+      setName("");
+    }
+  };
+
+  const handleDeleteTrainee = (index) => {
+    const updatedTrainees = [...trainees];
+    updatedTrainees.splice(index, 1);
+    setTrainees(updatedTrainees);
+  };
+
+  return (
+    <div>
+      <form onSubmit={handleAddTrainee}>
+        <input
+          value={name}
+          placeholder="Enter the name"
+          onChange={handleNameChange}
+        />
+        <button type="submit">Add a trainee</button>
+      </form>
```

Comment:
I think the add trainee was meant to be a link to a new page where you enter name, github username and select cohort